CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
+
+### Feature
+
+- Integrate support for param reactive expressions and expose `pn.rx` ([#5138](https://github.com/holoviz/panel/pull/5138), [#5582](https://github.com/holoviz/panel/pull/5582))
+- Implement `ChatMessage`, `ChatFeed` and `ChatInterface` components ([#5333](https://github.com/holoviz/panel/pull/5333))
+- Unify OAuth implementations and refresh `access_token` ([#5627](https://github.com/holoviz/panel/pull/5627))
+- Add `ColorMap` widget ([#5647](https://github.com/holoviz/panel/pull/5647))
+
+### Enhancement
+
+- Add unit to widget in `HoloViews` pane if provided ([#5535](https://github.com/holoviz/panel/pull/5535))
+- Allow registering global on_`session_destroyed` callback ([#5585](https://github.com/holoviz/panel/pull/5585))
+- Implement `auto_grow` on `TextAreaInput` ([#5592](https://github.com/holoviz/panel/pull/5592))
+- Add ability to redirect users from authorization callback ([#5594](https://github.com/holoviz/panel/pull/5594))
+- Add support for `Path` object in `FileDownload` ([#5607](https://github.com/holoviz/panel/pull/5607))
+- Add authorization_code and password based OAuth login handlers ([#5547](https://github.com/holoviz/panel/pull/5547))
+- Add format to `EditableFloatSlider` and `EditableIntSlider` ([#5631](https://github.com/holoviz/panel/pull/5631))
+- Add support for decorating async functions with `pn.io.cache` ([#5649](https://github.com/holoviz/panel/pull/5649))
+- Map `param.Bytes` to `FileInput` widget ([#5665](https://github.com/holoviz/panel/pull/5665))
+
+### Bug fixes
+
+- Fixes for `Column` invisible `scroll_button` taking space ([#5532](https://github.com/holoviz/panel/pull/5532))
+- Guard undefined values from being set on `BrowserInfo` ([#5588](https://github.com/holoviz/panel/pull/5588))
+- Fix thumbnails and use Panel design on index page ([#5595](https://github.com/holoviz/panel/pull/5595))
+- Fix regressions in `TextEditor` caused by migration to shadow DOM ([#5609](https://github.com/holoviz/panel/pull/5609))
+- Sync `location` state from request ([#5581](https://github.com/holoviz/panel/pull/5581))
+- Fix `Select` widget label offset in Material Design ([#5639](https://github.com/holoviz/panel/pull/5639))
+- Override token contents when reusing sessions ([#5640](https://github.com/holoviz/panel/pull/5640))
+- Fix patching a table with a `DataFrame` with a custom index ([#5645](https://github.com/holoviz/panel/pull/5645))
+- Set `FloatPanel` status correctly on initialization ([#5651](https://github.com/holoviz/panel/pull/5651))
+- Fix patching table with `pd.Timestamp` values ([#5650](https://github.com/holoviz/panel/pull/5650))
+- Ensure `notifications` and `browser_info` are loaded when `HoloViews` is loaded
```

Comment:
Should this line link to #5657 ?

Also: version number in merge request: "1.13.0" -> "1.3.0"

Reply:
Good catch, thanks!